cog/game.py

```
@@ -37,12 +37,12 @@ def get_channels():
 class Game(commands.Cog):
     # User can use this command to play ✊-🤚-✌️ with the bot in the command channel
     @discord.slash_command(name="rock_paper_scissors", description="玩剪刀石頭布")
-    # useser can choose ✊, 🤚, or ✌️ in their command
+    # user can choose ✊, 🤚, or ✌️ in their command
     async def rock_paper_scissors(
```

Ruff (0.8.2)

Missing return type annotation for public function `rock_paper_scissors`

Add return type annotation: `None`

(ANN201)